SMLHelper/Handlers/EatableHandler.cs

```diff
@@ -8,6 +8,7 @@
     /// </summary>
     public class EatableHandler : IEatableHandler
     {
+        private static bool pickupablePatched = false;
```

Comment:
Looks like this is never assigned. Move it to the patcher and set it to `true` after patching.
The handler should only call `PickupablePatcher.Patch` and have no business in determining if it's patched yet or not.